lib/twom.c

```diff
@@ -2962,7 +2962,16 @@ static int twom_txn_dump(struct twom_txn *txn, int detail)
             printf("COMMIT start=%08llX\n", (LLU)NEXT0(ptr, 0));
         }
         else if (type == DELETE) {
-            printf("DELETE ancestor=%08llX\n", (LLU)NEXT0(ptr, 0));
+            size_t parent_offset = NEXT0(ptr, 0);
+            const char *key = KEYPTR(loc->file->base + parent_offset);
+            int len = KEYLEN(loc->file->base + parent_offset);
```

Comment:
Signed integer overflow is undefined behavior. In practice, we could expect len to become negative if KEYLEN returns a very lange length, and the conditional in the next line won't catch that. Better to avoid either.

Reply:
yeah ,OK - it should be a size_t.  I just copied this from the other one nearby, so I'll fix them all.